test.js

```diff
@@ -36,7 +36,6 @@ class Test {
 
 async function test(context, testData) {
     logger.log("test started");
-    logger.log("testData: " + utils.stringify(testData));

```

Comment:
Moved this log to the failure case in line 54 because if the init didn't fail the test data will be logged from testConversation() function (line 118)